src/classes/search-podcast-result.ts

```diff
@@ -0,0 +1,24 @@
+import dlv from 'dlv';
+import { Podcast } from "../interfaces/podcast"
+import { getTagPageUriComponent } from '../utils/get-tag-page-uri-component';
+
+export class SearchPodcastResult implements Podcast {
+    description: string;
```

Comment:
sorry for going back on this description can be non mandatory. Also we don't need raw description at all.

Reply:
this will throw a error after my PR, only change you need to change it to IPodcast, just want to let you know.